src/components/Dropdown.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { PropTypes } from 'prop-types';
+
+class Dropdown extends React.Component {
+  state = {
+    displayMenu: false
+  };
+
+  showDropdownMenu = (event) => {
+    event.preventDefault();
+    this.setState({ displayMenu: true }, () => {
+      document.addEventListener('click', this.hideDropdownMenu);
+    });
+  };
+
+  hideDropdownMenu = () => {
+    this.setState({ displayMenu: false }, () => {
+      document.removeEventListener('click', this.hideDropdownMenu);
```

Comment:
Why are we adding and removing an eventListener?

Reply:
I added the Event Listener for `showDropdownMenu` so that when the dropdown is displayed it listen for any click, and closes the dropdown, 
For the `removeEvent Listener`  for `hideDropdownMenu` when the user click on the `fa-caret-down`  it closes the dropdowm

Reply:
https://github.com/andela/indiana-ah-frontend/pull/36/files#r269080868